src/app/(auth)/login/_types/login.interface.ts

```diff
@@ -22,3 +16,14 @@ export interface ILoginResponse {
   accessToken: string;
   refreshToken: string;
 }
+
+type ErrorKeys = 'email' | 'password';
+
+export interface IErrorResponse {
+  details: {
+    [key in ErrorKeys]: {
+      message: string;
+    };
+  };
+  message: string;
+}
```

Comment:
```suggestion
export interface IErrorResponse {
  details: {
    [key in keyof ILogin]: {
      message: string;
    };
  };
  message: string;
}
```

이런식으로 위에서 사용했던 타입을 재사용해도 괜찮을거같아요!

Reply:
와우와우 감사합니당